test/e2e/ClientMode.test.js

```diff
@@ -54,17 +54,29 @@ describe('clientMode', () => {
               res.push(_text);
             });
 
-            setTimeout(() => {
+            page.waitFor(3000).then(() => {
```

Comment:
Magic number, let's create file `test/server/helpers/server-constans.js` and move this values in variable with name